labs/02/feature_engineering.py

```diff
@@ -36,14 +36,14 @@ def main(args: argparse.Namespace) -> tuple[np.ndarray, np.ndarray]:
     # - for the rest of the columns, normalize their values so that they
     #   have mean 0 and variance 1; use `sklearn.preprocessing.StandardScaler`.
     #
-    # In the output, there should be first all the one-hot categorical features,
+    # In the output, firstly there should be all the one-hot categorical features,
```

Comment:
I suggest to revert this - this should be definitely "first".
See https://www.merriam-webster.com/words-at-play/first-or-firstly

Reply:
I actually immediately changed it back to `first` in f8f9794610.

For the typos/rewordings, I do not use the whole review process, and I do not ask students to update the pull request -- instead I merge the original pull request and then update it in a following commit (I understand the pull request mostly as an information that something is wrong, but if I do not agree with the solution, I shamelessly update it :-)